pages/ucsc/index.jsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
```

Comment:
should be replaced w/ a GraphPage (unless we want this as a demo of instant search or something)